contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--4.1.0--4.2.0.sql

```diff
@@ -862,5 +862,10 @@ CREATE OPERATOR CLASS sys.numeric_int8 FOR TYPE int8
 END IF;
 END $$;
 
+DROP CAST IF EXISTS(sys.BBF_VARBINARY AS sys.VARCHAR);
```

Comment:
this is problematic as dropping case could fail due to dependent objects and result in upgrade failure. Please talk to Dipesh regarding this.